### PR TITLE
feature: added grid

### PR DESCRIPTION
**Pull request by omgaikwad**
feature: added grid

### CODE REVIEW

File: components/grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.grid-two-equal-container {
+  width: 100%;
+  height: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
```
Comment (junaidshaikh-js) [inline components/grid/grid.css]: Instead, you can use only two columns as the name suggest. 

There won't be any need to add grid areas

File: components/grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  grid-template-areas: "container1 container2 container3 container4";
+}
+
+.grid-wrapper {
+  height: 14rem;
+}
+
+.grid-container-1 {
+  grid-area: container1;
+  box-sizing: border-box;
```
Comment (AnuragSahu11) [inline components/grid/grid.css:29]: You can declare all elements to border box at the start of css , just a suggestion btw

File: components/grid/grid.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
+.grid-two-equal-container {
+  width: 100%;
+  height: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-areas: "container1 container1 container2 container2";
+}
+.grid-three-equal-container {
```
Comment (junaidshaikh-js) [inline components/grid/grid.css]: similarly here. 

Also, you can rename to grid-three-column

**Review by AnuragSahu11 — COMMENTED** [review]
Keep up the good work

File: components/grid/grid.css [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+.grid-four-equal-container {
+  width: 100%;
+  height: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-areas: "container1 container2 container3 container4";
+}
+
+.grid-wrapper {
+  height: 14rem;
```
Comment (junaidshaikh-js) [inline components/grid/grid.css:24]: always add `min-height` unless you are using overflow-scroll.

**Review by junaidshaikh-js — COMMENTED** [review]
Good work!

File: components/grid/grid.css (hunk 1 above)
Comment (omgaikwad) [reply components/grid/grid.css]: Sure, I'll look into it

File: components/grid/grid.css (hunk 2 above)
Comment (omgaikwad) [reply components/grid/grid.css:29]: Thanks for the suggestion.

File: components/grid/grid.css (hunk 4 above)
Comment (omgaikwad) [reply components/grid/grid.css:24]: I did'nt thought about it, this is very valuable suggestion, I will make changes

**Pull request merged**